LandingPage.js

```diff
@@ -0,0 +1,69 @@
+import styles from './LandingPage.module.css';
+import classNames from 'classnames';
+
+function LandingPage() {
+    return (
+        <div>
+            <header>
+                <div className={styles.navBar}>
+                    <a href="#home">HOME</a>

```

Comment:
Instead of a tags for your routes to different parts of your application you should use Link from React Router. You will have to npm install react router dom first before you are able to utilize it throughout your app.

https://reactrouter.com/en/main/components/link